template/codeland.rb

```diff
@@ -176,10 +181,10 @@ def user_not_authorized
 end
 
 # Front-end stuffs
-
 gem 'autoprefixer-rails'
 gem 'bootstrap-sass', '~> 3.3.3'
 gem 'font-awesome-rails'
+gem 'draper', '~> 1.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.